src/utils/ctl/cart_ctl.c

```diff
@@ -270,54 +285,44 @@ print_usage_msg(const char *msg)
 	printf("-m 'log_message'\n");
 	printf("\tSpecify log message to be sent to remote server\n");
 	printf("--use_daos_agent_env\n");
-	printf("\tSet OFI and CRT_* vars through daos_agent\n");
+	printf("\tSet OFI and CRT_* vars through daos_agent\n\n");
 }
 
 static int
 parse_args(int argc, char **argv)
 {
-	int		option_index = 0;
-	int		opt;
-	int		rc = 0;
+	int	option_index = 0;
+	int	opt;
+	int	rc = 0;
 
 	ctl_gdata.cg_use_daos_agent_env = false;
 
 	if (argc <= 2) {
-		print_usage_msg("Wrong number of args\n");
+		print_usage_msg(NULL);
 		D_GOTO(out, rc = -DER_INVAL);
 	}
 
```

Comment:
would probably be good to consider using getopt() here so that you end up having a switch case of options instead of having a long if (strcmp(...)) else if (strcmp()) ...

Reply:
yeah, there are few more bigger changes i want to do in cart_ctl, but leaving those for the next patch